static/js/stripe-button.js

```diff
@@ -7,7 +7,6 @@ $(document).ready(function(){
     var handler = StripeCheckout.configure({
       key: 'pk_test_mlBqEQWc5FR1hGVCPuBH1kgy',
```

Comment:
Never commit any credentials to version control :) Please regenerate this key and use an environment variable for the handler.

Make an endpoint in flask and request the credential you need from there. 

Something like this maybe: 
```
@app.route('/credentials')
def credential_return(cred_type):
    if cred_type = "stripe-client-side-key":
        return os.environ.get('STRIPE-CLIENT-KEY')
```